deep_speech_2/data_utils/data.py

```diff
@@ -64,7 +68,6 @@ def __init__(self,
                  window_ms=20.0,
                  max_freq=None,
                  specgram_type='linear',
-                 use_dB_normalization=True,
```

Comment:
I didn't mean to remove `use_dB_normalization`. Maybe i made a mistake on git merging?